File: Iniciante/1066 - Pares, Ímpares, Positivos e Negativos/JavaScript/1066.js
function main() {
  const input = require("fs").readFileSync(0, "utf8").trim().split("\n");

  const NUMBER_OF_VALUES = 5;

  let even = 0;
  let odd = 0;
  let positive = 0;
  let negative = 0;

  for (let i = 0; i < NUMBER_OF_VALUES; i++) {
    const value = Number(input[i]);

    if (value % 2 === 0) {
      even++;
    } else {
      odd++;
    }

    if (value > 0) {
      positive++;
    } else if (value < 0) {
      negative++;
    }
  }

  console.log(`${even} valor(es) par(es)`);
  console.log(`${odd} valor(es) impar(es)`);
  console.log(`${positive} valor(es) positivo(s)`);
  console.log(`${negative} valor(es) negativo(s)`);
}

main();

File: Iniciante/1066 - Pares, Ímpares, Positivos e Negativos/Python/1066.py
def main():
    NUMBER_OF_VALUES = 5
    even = 0
    odd = 0
    positive = 0
    negative = 0

    for _ in range(NUMBER_OF_VALUES):
        value = int(input())

        if value % 2 == 0:
            even += 1
        else:
            odd += 1

        if value > 0:
            positive += 1
        elif value < 0:
            negative += 1

    print(f"{even} valor(es) par(es)")
    print(f"{odd} valor(es) impar(es)")
    print(f"{positive} valor(es) positivo(s)")
    print(f"{negative} valor(es) negativo(s)")


if __name__ == "__main__":
    main()
